index.html

```diff
@@ -0,0 +1,1091 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>New York Times</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <nav class="first-nav">
+    <div class="menu-icon">
+      <button type="submit">&#9776;</button>
```

Comment:
- [ ] **MANDATORY** You are failing your linter errors, this special characters often trigger them. As a suggestion, I would recommend you to use [Font awesome](https://fontawesome.com/v4.7.0/icon/bars) for your icons, or use a svg/image :)

Reply:
sorry I just pushed the file again I will update the demo link